From c847ee204e4a4a87f4e8074c72033545182190a3 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Wed, 29 Aug 2018 15:16:54 +0800
Subject: [PATCH 35/35] arm: dts: am335x-evm: enable wlan by mmc3

This commit comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/boot/dts/am335x-evm.dts |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/am335x-evm.dts b/arch/arm/boot/dts/am335x-evm.dts
index 7527dbf..e6939c4 100644
--- a/arch/arm/boot/dts/am335x-evm.dts
+++ b/arch/arm/boot/dts/am335x-evm.dts
@@ -768,7 +768,7 @@
 	dmas = <&edma_xbar 12 0 1
 		&edma_xbar 13 0 2>;
 	dma-names = "tx", "rx";
-	status = "disabled";
+	status = "okay";
 	vmmc-supply = <&wlan_en_reg>;
 	bus-width = <4>;
 	pinctrl-names = "default";
@@ -781,10 +781,11 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 	wlcore: wlcore@0 {
-		compatible = "ti,wl1835";
+		compatible = "ti,wl1835","ti,wl1271";
 		reg = <2>;
 		interrupt-parent = <&gpio3>;
 		interrupts = <17 IRQ_TYPE_LEVEL_HIGH>;
+		ref-clock-frequency = <38400000>;
 	};
 };
 
-- 
1.7.9.5

